build.gradle.kts

```diff
@@ -28,8 +28,8 @@ allprojects {
     version = "0.16.0-SNAPSHOT"
 }
 
-val publishedPluginsVersion by extra { "0.15.4" }
-val futurePluginsVersion = "0.15.5"
+val publishedPluginsVersion by extra { "0.15.5" }
```

Comment:
That means you already published the new versions?

Reply:
Yes